examples/Marks/Graph.ipynb

```diff
@@ -197,56 +244,108 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
    },
    "outputs": [],
    "source": [
     "tooltip = Tooltip(fields=['label', 'foo'], formats=['', '', ''])\n",
     "graph.tooltip = tooltip"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "### Custom styling of hovered and selected nodes"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
    "outputs": [],
    "source": [
-    "import bqplot.pyplot as plt\n",
-    "plt.clear()\n",
-    "line = plt.plot(np.cumsum(np.random.randn(20)))\n",
-    "# hover on nodes to see the plot\n",
-    "graph.tooltip = plt.current_figure()"
+    "graph.hovered_style = {'stroke': 'red'}\n",
+    "graph.unhovered_style = {'opacity': '0.4'}\n",
+    "\n",
+    "graph.selected_style = {'opacity': '1', 'stroke': 'white', 'stroke-width': '2.5'}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "# click on any node to select it (selected attribute gets populated with the index of the selected node)\n",
+    "graph.selected"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
    "source": [
-    "### Custom styling of hovered and selected nodes"
+    "### Add element click and background click handlers"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
    },
    "outputs": [],
    "source": [
-    "graph.hovered_style = {'stroke': 'red'}\n",
-    "graph.unhovered_style = {'opacity': '0.4'}\n",
+    "def print_event(self, target):\n",
+    "    print(target)\n",
     "\n",
-    "graph.selected_style = {'opacity': '1', 'stroke': 'white', 'stroke-width': '2.5'}"
+    "graph.on_background_click(print_event)\n",
+    "graph.on_element_click(print_event)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "### Add a line chart as a tooltip"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
    "outputs": [],
    "source": [
-    "# click on any node to select it (selected attribute gets populated with the selected node)\n",
-    "graph.selected"
+    "import bqplot.pyplot as plt\n",
```

Comment:
I think we can move this to the cell right after the tooltip. I think that feels better